src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -61,4 +63,35 @@ public WebDriverWait getWait() {
     public void setWait(final WebDriverWait wait) {
         this.wait = wait;
     }
+
+    /**
+     * Checks whether the specified file is downloaded or not by iterating 8 times where each iteration has a wait of 5 seconds.
+     * @param fileName expected downloaded file name
+     * @throws FileNotFoundException if the expected file not found
+     */
+    public void checkIfFileIsDownloaded(final String fileName)throws FileNotFoundException {
+        String downloadFileName = null;
+        boolean fileFound = false;
+        final long sleepTime = 5 * 1000;
+        int retryCount = 0;
+        while (!fileFound && retryCount <= MAX_RETRYCOUNT) {
+            File[] files = Hooks.getDownloadDirectory().listFiles();
+            for (File file : files) {
+                downloadFileName = file.getName();
+                if (downloadFileName.startsWith(fileName)) {
+                    fileFound = true;
+                    break;
+                }
+                try {
```

Comment:
I think this try-catch block sleeps between checking each file. Shouldn't it be moved out of the for loop to be part of the while?